### ISSUE TITLE
Doesn't fix empty strings after sorting

### ISSUE THREAD

**Issue by mishanga** [hudochenkov/stylelint-order#95]
Before autofix:
```css
.Divider {
    height: 100%;

    width: 100%;
}
```

After autofix:
```css
.Divider {

    width: 100%;
    height: 100%;
}
```

Config:
```js
module.exports = {
    "plugins": [
        "stylelint-order",
    ],
    "rules": {
        "order/properties-order": [
            {
                "emptyLineBefore": "always",
                "properties": [
                    "width",
                    "height",
                ],
            },
        ],
    },
};
```

Versions:
```
stylelint@9.10.1
stylelint-order@2.2.1
```

**Comment by shaikme** [comment]
this bug has 2 cases:
1) When stylelint remove property from first position
<img width="461" alt="Снимок экрана 2019-08-20 в 12 57 41" src="https://user-images.githubusercontent.com/32684639/63341081-18f53080-c351-11e9-8a33-9ee24178fc38.png">

2) When stylelint insert property to first position
<img width="495" alt="Снимок экрана 2019-08-20 в 13 05 02" src="https://user-images.githubusercontent.com/32684639/63341073-14c91300-c351-11e9-9296-b487c920a543.png">

**Comment by hudochenkov** [comment]
stylelint-order preserve empty lines before properties (and other nodes) unless `emptyLineBefore` and/or `noEmptyLineBetween` are specified. What happens in @mishanga case, that `width` had empty line before it, and it was preserved, because when property is the first in rule plugin doesn't change empty lines.

**Comment by shaikme** [comment]
@hudochenkov in our case `emptyLineBefore` and  `noEmptyLineBetween` specified, do you know how to avoid emty lines in this case?

**Comment by hudochenkov** [comment]
Improvement released in `stylelint-order@4.0.0`. Happy New Year!
